test/tests/postprocessors/pseudotimestep/tests

```diff
@@ -0,0 +1,22 @@
+[Tests]
+  design = 'PseudoTimestep.md'
+  issues = '#25931'
+  [pseudotimestepSER]
+    type = 'CSVDiff'
+    input = 'fv_burgers_SER.i'
+    csvdiff = 'fv_burgers_SER_out.csv'
+    requirement = 'This tests the pseudotimestepper postprocessor for the SER method.'
+  []
+  [pseudotimestepRDM]
+    type = 'CSVDiff'
+    input = 'fv_burgers_RDM.i'
```

Comment:
No need to create 3 input files. You could create and use the 

```
cli_args = "Postprocessors/pseudotimestap/method="RDM" Output/file_base=something"
```
Unless there are more parameters that change besides the parameters of the pseudo-timestepper.

Reply:
I know this trick but I was iffy if to use it since each method takes also different parameters for convergence, some are optional only for one method, and I feared it gets messy. But I give it a shot maybe it doesn't look too bad.

Reply:
Changed to this, but I left one file untouched for the .md